docs/style-guide.rst

```diff
@@ -757,45 +757,50 @@ No::
 
     pragma solidity >=0.4.0 <0.7.0;
 
+
     // Base contracts just to make this compile
     contract B {
         constructor(uint) public {
         }
     }
+
+
     contract C {
         constructor(uint, uint) public {
         }
     }
+
+
     contract D {
         constructor(uint) public {
         }
     }
 
+
     contract A is B, C, D {
         uint x;
 
         constructor(uint param1, uint param2, uint param3, uint param4, uint param5)
         B(param1)
         C(param2, param3)
-        D(param4)
-        public
-        {
+        D(param4) {
             x = param5;
         }
     }
 
+
     contract X is B, C, D {
         uint x;
 
         constructor(uint param1, uint param2, uint param3, uint param4, uint param5)
             B(param1)
             C(param2, param3)
-            D(param4)
-            public {
```

Comment:
Typo? (`public` being removed)

Reply:
Sorry, I was told to remove a lot I had added after solhint recommendations, and got a bit keen removing `public's I never added…